### ISSUE TITLE
Visualize the Playground AST

### ISSUE THREAD

**Issue by emilkrebs** [eclipse-langium/langium-website#204]
Visualize this like the DomainModel example with interactivity:
![image](https://github.com/eclipse-langium/langium-website/assets/68400102/6935e7e6-6866-407e-9a6b-73780144dd7d)

**Comment by spoenemann** [comment]
I would prefer to keep the JSON-like syntax because it's more precise and it can be copied into a JS/TS file to test something with it. I don't see much value in a graphical representation in this case.

**Comment by montymxb** [comment]
I would think we would keep the JSON syntax, but also provide an additional context to view the AST structure that is not just textual.

I would suggest another way to look at this as well. The value lies in garnering more interest around Langium & ASTs by providing some way other than JSON to inspect the results. Especially in demonstrating alternative ways that ASTs can be understood & observed for those who are not familiar with them (consider newcomers to Langium or DSL design in general). The intended audience here is not just experts, who are seeking textual tools, but newcomers who are seeking additional aides (like visual aides) to help learn more about this field. In this context, I think this could make a worthwhile addition, and is least worth investigating for this purpose.

**Comment by spoenemann** [comment]
Ok if it's an optional view that is added on demand we could do that.

**Comment by montymxb** [comment]
Yep, that's the plan, as the existing output is very important to retain. In the end we can observe how this supplemental approach works in practice, and see if it makes sense to continue in this direction.
